AEPAssurance/Source/Assurance.swift

```diff
@@ -28,7 +28,6 @@ public class Assurance: NSObject, Extension {
 
     let datastore = NamedCollectionDataStore(name: AssuranceConstants.EXTENSION_NAME)
     var assuranceSession: AssuranceSession?
-    var shouldProcessEvents: Bool = true
```

Comment:
renamed shouldProcessEvents to `canProcessSDKEvents` same in Android
and moved it to AssuranceSession class. Looks more like it should belong there